### ISSUE TITLE
Use the default split instead of split(" ") in prepared_text function

### ISSUE THREAD

**Issue by jmcrey** [a-alwu/pwp-capstones#1]
```
import re 
def prepare_text(text):
    regex = r"[.,'!?\"]"  
    prepared_text = re.sub(regex, "", text, 0).lower().split(" ")
    return prepared_text
```

This is a really great function for getting rid of all unneeded values from the string. However, we should use the `split()` syntax function instead of `split(" ")` because `split()` gets _all_ whitespace characters -- not just a singular space. For example, consider a sentence like "Hello?    How are you?" -- `split()` will separate by all the white space; but, `split(" ")` will only separate by one space.

So, basically, for this to consider all possible white space scenarios we just have to change the line

`prepared_text = re.sub(regex, "", text, 0).lower().split(" ")`

to

`prepared_text = re.sub(regex, "", text, 0).lower().split()`

But, overall, this is a really well done function! I especially like the fact that we're using the `re` library to substitute all those letters in the text!
